### ISSUE TITLE
focalloss

### ISSUE THREAD

**Issue by shuaizzZ** [Tramac/awesome-semantic-segmentation-pytorch#54]
Have you tried the effect of focalloss on semantic segmentation? I tried to implement focalloss myself but the effect is worse than cross entropy. Can you add focalloss to your loss function?

**Comment by Tramac** [comment]
Focal loss is effective for small targets. There is my [Tensorflow implementation](https://github.com/Tramac/CHAOS/blob/f127d306abe2f1bc4800e268da5dfdcc0c1fb2e4/utils/losses.py#L157), maybe it helps you.
